### PR TITLE
fix invites

### PR DESCRIPTION
**Pull request by mattkrick**
refactor notifications, too

### CODE REVIEW

**Comment by mattkrick** [comment]
@jordanh feel free to merge while that coffee is a brewin.

File: src/universal/modules/notifications/ducks/notifications.js [hunk 1]
```diff
@@ -1,51 +1,58 @@
...
 
-export function show(opts, level = 'success') {
+let nid = 0;
+export function show(opts, level = SUCCESS) {
   return {
     type: NOTIFICATIONS_SHOW,
-    ...opts,
-    level,
-    nid: ++nid
+    payload: {
```
Comment (jordanh) [inline src/universal/modules/notifications/ducks/notifications.js]: 👏

File: src/universal/modules/invitation/containers/Invitation/Invitation.js [hunk 2]
```diff
@@ -3,21 +3,74 @@ import {connect} from 'react-redux';
...
 } from 'universal/modules/notifications/ducks/notifications';
 
+import {
+  invalidInvitation,
+  inviteNotFound,
+  teamAlreadyJoined,
+  successfulJoin
+} from 'universal/modules/invitation/helpers/notifications';
+
+const getUserWithMemberships = `
```
Comment (jordanh) [inline src/universal/modules/invitation/containers/Invitation/Invitation.js]: Oh, there was more here to do than met the 👀 . Thank you!

File: src/universal/containers/Logout/LogoutContainer.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React, {PropTypes, Component} from 'react'; // eslint-disable-line no-unused-vars
 import {connect} from 'react-redux';
-import {show} from 'universal/modules/notifications/ducks/notifications';
+import {showSuccess} from 'universal/modules/notifications/ducks/notifications';
```
Comment (jordanh) [inline src/universal/containers/Logout/LogoutContainer.js]: Yes, I think this was as good as time as any to do this. Thank you!

**Comment by jordanh** [comment]
Wow man, awesome!!

**Pull request merged**